q2_sample_classifier/plugin_setup.py

```diff
@@ -83,7 +85,9 @@
     'contain overlapping features with the feature table used to train '
     'the estimator.')
 
-inputs = {'table': FeatureTable[Frequency]}
+inputs = {'table': FeatureTable[Frequency | RelativeFrequency |
```

Comment:
This `inputs` gets used in a bunch of actions registered below, I was pretty sure we only want to enact these changes on `split_table`.